### ISSUE TITLE
Getting tons of Trakt error messages on Kodi

### ISSUE THREAD

**Issue by pinkpajamarama** [Sanqui/discard2#32]
I installed the latest version of kodi.   Since then I keep getting multiple Trakt error messages although it still works.  Trakt said it was a kodi problem.   The messages I'm getting are:  Trakt error 502,  Remote communication server failed to start and Trakt wait limit reached 429.    I am mainly using scrubs v2.   Anyone?   

I don't do builds and I"m a newbie so please dumb it down for me.  Thanks!

**Comment by TheTechRobo** [comment]
Uh, this isn't Kodi. I'm also not sure what Trakt is.

**Comment by pinkpajamarama** [comment]
I got this link from Trakt to fix the problem. Trakt is an app that keeps track of what you're watching;  Google it.
